samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -13,13 +13,14 @@
 
```

Comment:
For the commit `samples/sockets: Fix native app detection when selecting headers`, I suggest to change the subject to
`samples/net/sockets: ....`
so that the searching networking commits gets slightly easier.

Reply:
Sounds good. Fixed.